### DISCUSSION TITLE
How to implementation set more than 4 weights with CPU?

### DISCUSSION THREAD

**Discussion by will4621d** [google/filament#4981] [Q&A]
Hi all, I'm developing an virtual avatar in Android, in which models have 52 blendshapes, whereas I can only change 4 of them with RenderableManager.setMorphWeights(). In relavant issues I saw some using CPU to achieve this, but I'm not familiar with C, anyone can tell me how to figure it out?

**Comment by romainguy** [comment]
We're investigating solutions to fix this limitation. @pixelflinger @bejado

**Comment by bejado** [comment]
See https://github.com/google/filament/issues/4772.
